dev/io.openliberty.wsoc.internal_fat/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+* Copyright (c) 2020 IBM Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+dependencies {
+  requiredLibs 'net.sf.jtidy:jtidy:9.3.8',
+    'org.eclipse.jetty.websocket:websocket-client:9.2.2.v20140723',
+    'org.eclipse.jetty.websocket:websocket-api:9.2.2.v20140723',
+    'org.eclipse.jetty.websocket:websocket-common:9.2.2.v20140723',
+    'javax.websocket:javax.websocket-api:1.0',
+    'commons-codec:commons-codec:1.11',
+    'org.eclipse.jetty.websocket:javax-websocket-client-impl:9.2.2.v20140723',
+    'org.eclipse.jetty:jetty-io:9.2.2.v20140723',
+    'org.eclipse.jetty:jetty-util:9.2.2.v20140723',
+    'org.apache.httpcomponents:httpclient:4.5.4',
+    'commons-logging:commons-logging:1.1.3'
+}
+
+addRequiredLibraries {
+  doLast {
+    // Define dependency groups
+    configurations {
+      junit
+    }
+    // Define G:A:V coordinates of each dependency
+    dependencies {
+      junit 'junit:junit:4.12'
+    }
+    // Copy the dependencies wherever they may be needed
+    copy {
+      // Copy junit jars into a specific folder so they can be referenced
+      from configurations.junit
+      into "${buildDir}/autoFVT/test-applications/basic.war/resources/WEB-INF/lib"
+    }
```

Comment:
Is this necessary? I think we get junit for free for any FAT buckets, or is there something specific here for the websocket tests?

Reply:
Yeah so we do get the junit file for free but inside the war app itself it needs the jar for some assertions it makes and other junit related functions. In CL it was already in the folder but we can't have that in OL so I removed it and add it when building the project and later the app gets build with it. If I just remove it and dont add it the test fail with a class not found exception